rules/windows/collection_email_powershell_exchange_mailbox.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/12/15"
+maturity = "production"
+updated_date = "2020/12/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of PowerShell Exchange New-MailBoxExportRequest Command to export content of a primary mailbox or archive
+to a .pst file. Adversaries may target user email to collect sensitive information.
+"""
+false_positives = ["Legitimate exchange system administration activity."]
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Exporting Exchange MailBox via PowerShell"
+references = [
+    "https://www.volexity.com/blog/2020/12/14/dark-halo-leverages-solarwinds-compromise-to-breach-organizations/",
+    "https://docs.microsoft.com/en-us/powershell/module/exchange/new-mailboxexportrequest?view=exchange-ps",
+]
+risk_score = 47
+rule_id = "6aace640-e631-4870-ba8e-5fdda09325db"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Collection"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  process.name: "powershell.exe" and process.args : "New-MailboxExportRequest*"
```

Comment:
Should there be a wildcard in front of the cmdlet (`New-MailboxExportRequest`)?

```
C:\Windows\system32\cmd.exe /C powershell.exe -PSConsoleFile exshell.psc1 -Command “New-MailboxExportRequest -Mailbox foobar@organization.here -ContentFilter {(Received -ge ’03/01/2020′)} -FilePath ‘\\<MAILSERVER>\c$\temp\b.pst'
```

Reply:
So I talked to Samir on this one, having a leading wildcard impacts performance which is why it's left off.  It does currently trigger without the leading wildcard.